images/kexpand/onbuild.sh

```diff
@@ -7,7 +7,10 @@ mkdir -p /go/src/github.com/kopeio
 ln -s /src/ /go/src/github.com/kopeio/kexpand
 
 cd /go/src/github.com/kopeio/kexpand
-make gocode
+if [ ! -d vendor ]; then
```

Comment:
added so a new user without experience using govendor can download the source and run `make image` without requiring a local installation of go just to retrieve libraries.